include/cadmium/engine/pdevs_simulator.hpp

```diff
@@ -53,6 +52,9 @@ namespace cadmium {
             TIME _last;
             TIME _next;
 
+            //logging purposes
+            std::string _model_id;
```

Comment:
Same as the coordinator, maybe const char* and getting the name from alternative sources. For example: in my last commit I included this in the clock example:
- inline static const char* pretty_name="HOUR GENERATOR";
then using if constexpr "has_pretty_name" I expect to read that name in place of generating one with boost::typeinfo

Reply:
Ok, sounds good, as you are working on this pretty_name logic I will wait to be ready and then implement this suggestion